flexmeasures/data/models/time_series.py

```diff
@@ -17,6 +20,9 @@
     exclude_source_type_filter,
 )
 from flexmeasures.data.services.time_series import collect_time_series_data
+from flexmeasures.ui.charts import chart_type_to_chart_specs
```

Comment:
Actually, it could then be called by `SensorUI.get_chart` right away which also feels cleaner. Because `SensorAPI` is supposed to only be about JSON.

Reply:
I agree with this policy of cleanly separating packages. But I don't think moving this to the UI package would completely resolve this. When we move GETting charts to our API package, the API package would then be importing from the UI module, which is also a situation that we try to avoid (the UI package importing from the API packaage is expected, though).

I have an alternative suggestion. Bare with me, because it might be counter intuitive. **I suggest to move the chart specs into the data package.** That is, let's perceive chart specs as just another data model, similar to forecasting and scheduling models. This way, when the API package needs access to chart specs, too, it will import from the data package rather than the UI package. I can imagine we'd then want `Sensor.chart()` to always return a (vega-lite compatible) dict instead of JSON or HTML, and let other packages convert it as needed (the API makes it into a JSON and the UI makes it into HTML, for example).

Reply:
I'm prepared to consider this in a review. Would that live in `data.models.charts`? Please add a good sentence or two describing why we consider these chart specs as data models.

Reply:
See https://github.com/SeitaBV/flexmeasures/pull/104